packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -22,14 +22,42 @@ import {
 import {commerceAPIConfig, einsteinAPIConfig} from '../../api.config'
 import {DEFAULT_LOCALE, DEFAULT_CURRENCY, urlPartPositions} from '../../constants'
 import {getPreferredCurrency, getSupportedLocalesIds} from '../../utils/locale'
-import {getUrlConfig} from '../../utils/utils'
+import {getUrlConfig, isServer} from '../../utils/utils'
 import {getAppOrigin} from 'pwa-kit-react-sdk/utils/url'
+import http from 'http'
+import https from 'https'
+
+const KEEP_ALIVE_MS = 8000
```

Comment:
I _think_ we'd like to keep connections warm for the life time of the Lambda / server.

I'd recommend we start this value at one minute.

```suggestion
const KEEP_ALIVE_MS = 60 * 1000
```